### ISSUE TITLE
Consider sorting utterances/comments newest to oldest

### ISSUE THREAD

**Issue by k-dominik** [bioimage-io/archive.bioimage.io#375]
Comments for models go from oldest to newest. Usually the story is: hey something is off with the model ... scroll scroll... fixed. Which might be a bit of a waste of time to read through. Also at first glance it will seem that many models are broken, given the oldest comment.

The severity of this is probably pretty low - only users logged into this utterances thing see comments at all.

**Comment by jmetz** [comment]
I think the order is correct, basing this on almost all chat-type systems I can think of including messages here in github issues.

I would instead suggest perhaps either auto-scrolling to the latest message (like many chat interfaces), or having a "scroll to latest" button.

**Comment by oeway** [comment]
I am thinking maybe we should wait for the new chat feature with the s3 upload workflow? @jmetz We can think about this when we have the json file for each model. What do you think?

**Comment by jmetz** [comment]
Yep I agree - and for first release, probably a "scroll up latest message" works well
